### ISSUE TITLE
[FEATURE] Hide/mute user without blocking them

### ISSUE THREAD

**Issue by stevenroose** [vitorpamplona/amethyst#570]
**Is your feature request related to a problem? Please describe.**
Many users show up in search result with posts I find garbage, but I don't want to prevent these people from showing up in comment threads or in comments in my own threads. 

**Describe the solution you'd like**
A hide/mute function (like Mastodon has) that hides the user from all timelines but not from comment threads. 

**Bounty (in Bitcoin sats) offered for the implementation**

**Additional context**

**Comment by OrigiSize** [comment]
To be clear you're saying the posts are not spam but are just not relevant to your search?

There doesn't appear to be a NIP that handles this distinction, therefore this feature would seem to require a local database.

**Comment by stevenroose** [comment]
Yeah. I think it makes sense to have a "timeline filter" of things (hashtags or people) you don't want to see while browsing. But you might not necessarily have these people cut out of conversations.
